writer.go

```diff
@@ -59,6 +59,11 @@ func (w *Writer) Write(b []byte) (int, error) {
 	return w.writeAndRetry(w.priority, string(b))
 }
 
+// RawWrite sends a log message with a custom priority
+func (w *Writer) RawWrite(p Priority, b []byte) (int, error) {
```

Comment:
`RawWrite` isn't the best name for this. Perhaps `WriteWithPriority` or something would be better.

Why does this take a `[]byte` when all the other write functions take a `string` (and this has to convert to a `string` anyway)?